src/stylesheets/core/_variables.scss

```diff
@@ -16,6 +16,7 @@ $lead-line-height:    1.7;
 
 $font-sans:           'Source Sans Pro', $helvetica;
 $font-serif:          'Merriweather', $georgia;
+$font-path:           '../fonts'
```

Comment:
You could keep what you have here in variables.scss although I think we'll get rid of that file at some point.